src/App.jsx

```diff
@@ -13,6 +14,7 @@ function App() {
         <Routes>
           <Route path="/" element={<Landing />} />
           <Route path="/editor" element={<Editor />} />
+          <Route path="/usermainpage" element={<UserMainPage />} />
```

Comment:
I suggest using a shorter name in the path, e.g., main, workspace, codespace.

```suggestion
          <Route path="/main" element={<UserMainPage />} />
```